src/test/scala/nl/hugo/redbook/ch6/CNG.scala

```diff
@@ -0,0 +1,24 @@
+package nl.hugo.redbook.ch6
+
+object CNG {
```

Comment:
Yes, it means exactly that. I don't think `RNG` is a stretch, but a perfectly accepted acronym. In the same spirit, I came up with the `CNG`. If you feel it's important, we could add a comment explaining the acronym, or you could rename it back to `RNGMock`, `MockRNG` or `NonRNG` if you feel strong about it.